test/pls.jl

```diff
@@ -276,9 +302,9 @@ end
     @test fmnc.θ == [fm.θ[1], fm.θ[3]]
     @test lowerbd(fmnc) == zeros(2)
 
-    fmnc = zerocorr!(LMM(@formula(reaction ~ 1 + days + (1+days|subj)), slp));
```

Comment:
do we test `zerocorr!` applied to a whole model anywhere now?

Reply:
I guess we should add such a test back in. 

An alternative is to drop the use of `zerocorr!` entirely in favor of `zerocorr` on individual terms.  It is always good when there are multiple ways of doing something to ask if there is anything to be gained from having said multiple ways.  @kleinschmidt Do you have any opinions?

Reply:
I think for most of the cases where someone might want to use `zerocorr!` they're probably going to have a small number of grouping factors with a lot of terms in each one, so there's not much to be gained from `zerocorr!`

Reply:
And post-hoc modification of fitted models (i.e. the RK approach)?

Reply:
To answer the original question by @palday yes we do test `zerocorr!` elsewhere - a few lines above this.  It is line 310 in the current numbering.

Let's leave this as it is and remind ourselves to revisit the question of whether to keep `zerocorr!` in the package at a later time.